lib/gooddata/lcm/actions/synchronize_users.rb

```diff
@@ -217,7 +221,7 @@ def call(params)
   We are unable to get the value to filter users."
                       end
 
-                      puts "Project #{project.pid} will receive #{filtered_users.count} from #{new_users.count} users"
+                      params.gdc_logger.info("Project #{project.pid} will receive #{filtered_users.count} from #{new_users.count} users")
```

Comment:
Similar code found in synchronize_users.rb:174, synchronize_users.rb:202, synchronize_users.rb:224, synchronize_users.rb:291 (mass = 64)